Dockerfile

```diff
@@ -29,6 +29,7 @@ RUN unzip /x/redigo.zip -d /x && unzip /x/snappy-go.zip -d /x && \
 
 # Build the local gddo files.
 ADD . /go/src/github.com/golang/gddo
+RUN go get google.golang.org/cloud/compute/metadata
 RUN go install github.com/golang/gddo/gddo-server
```

Comment:
For future-proofing, this line should be 'go get' and you can remove the line introduced in this PR.